### PR TITLE
(#227) Add additonal details to LDAP configuration documentation

### LINKED ISSUE #227: Add additional LDAP Configuration details
In addition to Email address, the Surnamae and GivenName fields must also not be empty.

### PR DESCRIPTION
**Pull request by steviecoaster**
Prior to this commit, we specified that only the
email address field would need to be filled in.

This commit addresses some additonal information that is also required.

Fixes #227

### CODE REVIEW

**Review by vexx32 — APPROVED** [review]

File: CentralManagement/CentralManagementSetupWeb.md [hunk 1]
```diff
@@ -10,43 +10,43 @@ ___
...
+    - [All attempts to send email from CCM result in an error](#all-attempts-to-send-email-from-ccm-result-in-an-error)
+    - [Emails sent from CCM to users has links that contains localhost, rather than actual CCM Server name](#emails-sent-from-ccm-to-users-has-links-that-contains-localhost-rather-than-actual-ccm-server-name)
+    - [The updated license file is not being picked up in the website](#the-updated-license-file-is-not-being-picked-up-in-the-website)
+    - [A computer or group is not showing as available for deployments but I have plenty of available licenses](#a-computer-or-group-is-not-showing-as-available-for-deployments-but-i-have-plenty-of-available-licenses)
+    - [Failed to generate a user instance of SQL Server due to failure in retrieving the user's local application data path.](#failed-to-generate-a-user-instance-of-sql-server-due-to-failure-in-retrieving-the-users-local-application-data-path)
+    - [The website app pool shuts down almost as soon as it is started](#the-website-app-pool-shuts-down-almost-as-soon-as-it-is-started)
```
Comment (vexx32) [inline CentralManagement/CentralManagementSetupWeb.md:49]: Not critical, but fix your dang markdown extension, Stevie!

**Review by gep13 — APPROVED** [review]
LGTM!

File: CentralManagement/CentralManagementSetupWeb.md (hunk 1 above)
Comment (gep13) [reply CentralManagement/CentralManagementSetupWeb.md]: @vexx32 I have taken the liberty of fixing this up.

**Comment by gep13** [comment]
@steviecoaster your changes have been merged, thanks for your contribution 👍

**Pull request merged**